modules/languages/php.nix

```
@@ -0,0 +1,100 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
+  cfg = config.vim.languages.php;
+
+  defaultServer = "phpactor";
+  servers = {
+    phpactor = {
+      package = pkgs.phpactor;
+      lspConfig = ''
+        lspconfig.phpactor.setup{
+          capabilities = capabilities,
+          on_attach = default_on_attach,
+          cmd = ${
+          if isList cfg.lsp.package
+          then nvim.lua.expToLua cfg.lsp.package
+          else ''
+            {
+              "${getExe cfg.lsp.package}",
+              "language-server"
+            },
+          ''
+        }
+        }
+      '';
+    };
+
+    phan = {
+      package = pkgs.php81Packages.phan;
+      lspConfig = ''
+        lspconfig.phan.setup{
+          capabilities = capabilities,
+          on_attach = default_on_attach,
+          cmd = ${
+          if isList cfg.lsp.package
+          then nvim.lua.expToLua cfg.lsp.package
+          else ''
+              {
+                "${getExe cfg.lsp.package}
```

typo lol missing `",` at the end
```suggestion
                "${getExe cfg.lsp.package}",
```

---

good catch